src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -618,21 +627,22 @@ private Compilation CreateEmptyCompilation()
             }
 
             private async Task<(Compilation compilationWithoutGenerators, Compilation? compilationWithGenerators, GeneratorDriver? generatorDriver)> BuildDeclarationCompilationFromInProgressAsync(
-                InProgressState state, Compilation compilationWithoutGeneratedDocuments, CancellationToken cancellationToken)
+                InProgressState state, CancellationToken cancellationToken)
```

Comment:
second parameter was redundant given that it is retrievable from `InProgressState`.